website/src/components/Showcase/Card.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import styles from '../../pages/showcase.module.css';
```

Comment:
Use individual module css. Why do you use shared css file for all components?

Reply:
Made sure to do this moving forward. My apologies for not splitting them up per component.